src/FlightBookingApp.API/appsettings.Development.json

```diff
@@ -20,7 +20,8 @@
     "Issuer": "https://localhost:7057",
     "Audience": "https://localhost:7057",
     "AccessTokenExpirationInMinutes": 10,
-    "RefreshTokenExpirationInDays": 60
+    "RefreshTokenExpirationInDays": 60,
```

Comment:
You have two different Expiration Time options. Why? This can lead to mistakes. Also, you mustn't make access token too long, it is insecure, 60 minutes is ok, but not 1 day